### DISCUSSION TITLE
在部署NoneBot1/2时均出现以下错误，求问如何解决

### DISCUSSION THREAD

**Discussion by NOTAKUlia** [nonebot/discussions#104] [问答]
![image](https://user-images.githubusercontent.com/62437056/154509001-2cf9f4af-16a0-406b-8e10-8b97618de475.png)
OS：winserver 2008 R2
python：3.8.10
在使用pip命令安装nb-cli时，出现如图上报错
在使用pip命令安装NoneBot时，出现如图上报错
在使用pip命令安装NoneBot2时，出现如图上报错
求问如何解决

**Comment by yanyongyu** [comment]
请使用64位的python或者更新一下pip和setuptools

**Reply by NOTAKUlia** [reply]
python为x64，且pip使用的是zuixin 22版本

**Reply by yanyongyu** [reply]
报错是pygtrie安装失败，setuptools是最新版本吗
